### PR TITLE
depthwise_conv2d_backprop_input

### LINKED ISSUE #1548: Add Neural Network functions for TensorFlow frontend
Add [Neural Network functions](https://www.tensorflow.org/api_docs/python/tf/nn#functions) for TensorFlow frontend:
- [ ] atrous_conv2d
- [ ] atrous_conv2d_transpose
- [ ] avg_pool1d
- [ ] avg_pool2d
- [ ] batch_normalization
- [ ] collapse_repeated
- [ ] compute_average_loss
- [ ] conv1d
- [ ] conv1d_transpose
- [ ] conv2d_transpose
- [ ] conv3d_transpose
- [ ] conv_transpose
- [ ] convolution
- [ ] crelu
- [ ] ctc_unique_labels
- [ ] depthwise_conv2d
- [ ] depthwise_conv2d_backprop_filter
- [ ] depthwise_conv2d_backprop_input
- [ ] dropout
- [ ] embedding_lookup
- [ ] embedding_lookup_sparse
- [ ] erosion2d
- [ ] gelu
- [ ] l2_normalize
- [ ] local_response_normalization
- [ ] log_poisson_loss
- [ ] max_pool1d
- [ ] max_pool2d
- [ ] moments
- [ ] nce_loss
- [ ] normalize_moments
- [ ] pool
- [ ] safe_embedding_lookup_sparse
- [ ] sampled_softmax_loss
- [ ] scale_regularization_loss
- [ ] separable_conv2d
- [ ] sigmoid_cross_entropy_with_logits
- [ ] silu
- [ ] sufficient_statistics
- [ ] swish
- [ ] weighted_cross_entropy_with_logits
- [ ] weighted_moments
- [ ] with_space_to_batch
- [ ] zero_fraction

### PR DESCRIPTION
**Pull request by Bhavit03**
Close #1548

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:
